src/main/java/mate/academy/bookstore/model/CartItem.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
```

Comment:
`@Table`

Reply:
Is it always necessary to annotate @Table? After all, the name of the class matches the name of the created table, once you wrote me that it is not necessary to do it.

Reply:
I'm afraid that I didn't told you that. 
According to the naming convention table name should be plural 
So we need to specify the name